vendor/extensions/announcements/app/views/refinery/announcements/admin/announcements/_form.html.erb

```diff
@@ -22,6 +22,11 @@
                 :object => "announcements/announcement" -%>
   </div>
 
+  <div class='field'>
+    <%= f.label :link -%>
```

Comment:
My first reaction to seeing a spot for a link was to put a full URL (I used google.com) as opposed to a subdirectory (like /reports/4). Maybe add an example to the label to illustrate it more?